src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comments } from '../../types/Comments';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comments;
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  comment: {
+    name,
+    email,
+    body,
+  },
+}) => (
```

Comment:
It's not a good way to destructuring. Look at my suggestion. It's much easier to read 
```suggestion
export const CommentInfo: React.FC<Props> = ({ comment }) => {
  const { name, email, body } = comment;
  
  return ( ...
```